packages/cosmos_ui_components/pubspec.lock

```diff
@@ -68,8 +68,8 @@ packages:
     dependency: "direct main"
     description:
       path: "packages/cosmos_utils"
-      ref: starport-redesign
-      resolved-ref: "5e7da9b618d69acebed284c2711d026acc1ca551"
+      ref: gradient_avatar
```

Comment:
@andrzejchm 
Added this because I added a function in `cosmos_utils`. Should have been a small separate PR but since we are already into nested `starport-redesign` Pas so I thought I should update the reference, will update it back to `starport-redesign` in the next PR